experimental/dds/tree/src/EditLog.ts

```diff
@@ -434,20 +526,15 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 			);
 		}
 
-		assert(index - this.numberOfSequencedEdits < this.localEdits.length, 'Edit to retrieve must be in the log.');
 		return this.localEdits[index - this.numberOfSequencedEdits];
 	}
 
 	/**
 	 * {@inheritDoc OrderedEditSet.tryGetEdit}
 	 */
 	public async tryGetEdit(editId: EditId): Promise<Edit<TChange> | undefined> {
-		try {
-			const index = this.getIndexOfId(editId);
-			return await this.getEditAtIndex(index);
-		} catch {
-			return undefined;
-		}
+		const index = this.tryGetIndexOfId(editId);
```

Comment:
I think the try/catch was to catch errors _downloading_ the chunk, which I think would now get thrown.
@jenn-le to double check.

Reply:
yes, I gree we should keep the try/catch

Reply:
Yes, exactly.